homeassistant/components/simplisafe/__init__.py

```diff
@@ -83,6 +92,7 @@
 ATTR_PIN_LABEL_OR_VALUE = "label_or_pin"
 ATTR_PIN_VALUE = "pin"
 ATTR_SYSTEM_ID = "system_id"
+ATTR_NOTIFICATION_TEXT = "notification_text"
```

Comment:
What do you think about using `ATTR_MESSAGE` for this constant? Or is this constant specific and known by users of SimpliSafe?
https://github.com/home-assistant/home-assistant/blob/0ae86b022d6416e5b061f010944fb5cdd77e6ed0/homeassistant/components/notify/__init__.py#L24-L25

Reply:
`message` will work!